src/_stylesheets/_variables.scss

```diff
@@ -1,2 +1,3 @@
 $app-light-blue: govuk-tint($govuk-brand-colour, 95%);
```

Comment:
Feels like this needs to be renamed to `$app-blue-tint-95` for consistency's sake, but that can be done separate from this PR.